src/Build/Logging/BinaryLogger/BinaryLogRecordKind.cs

```diff
@@ -39,7 +39,6 @@ public enum BinaryLogRecordKind
         String,
         TaskParameter,
         ResponseFileUsed,
-        GeneratedFileUsed,
```

Comment:
This was also a problematic change (new name for an existing value). Presumably it wasn't in main long enough for us to consider leaving it here.

Reply:
Correct.
Btw. this wouldn't be an issue for builds or logs, but for theoretically authored custom tooling or extension taking a dependency on the new value. I believe that coding against nightly build is not comming with 100% guarantee of compatibility - however we should finalize our thoughts before we are done with next preview (mainly of 17.10)